src/menus/PluginMenus.cpp

```diff
@@ -32,6 +32,383 @@
 // private helper classes and functions
 namespace {
 
+using MenuList = std::vector<std::pair<TranslatableString, std::vector<TranslatableString>>>;
+
+const MenuList& PreferredEffectsMenu()
+{
+   static MenuList items =
+   {
+      {
+         XO("Volume and compression"),
```

Comment:
Ah, sorry, you copied my typo here. It should be "Volume and Compression" with a big C